### PR TITLE
#2656 introduce docker volume to support add config operations

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2656)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-agent/src/main/scala/com/island/ohara/agent/docker/DockerClient.scala [hunk 1]
```diff
@@ -118,6 +118,45 @@ trait DockerClient extends Releasable {
...
+    */
+  def forceRemoveConfig(name: String): Boolean
+
+  /**
+    * get the config list pairs by specific name.
+    *
+    * @param name the config list name
+    * @return config list pairs
+    */
+  def inspectConfig(name: String): Map[String, String]
```
Comment (chia7712) [inline ohara-agent/src/main/scala/com/island/ohara/agent/docker/DockerClient.scala]: Why not making configs be a part of ContainerInfo?

Comment (saivirtue) [reply ohara-agent/src/main/scala/com/island/ohara/agent/docker/DockerClient.scala]: You mean introduce configs into containers?
This pr focus on adding a new docker method to Ohara.

**Comment by saivirtue** [comment]
retry sshConnector

**Comment by saivirtue** [comment]
retry manager-api

**Comment by saivirtue** [comment]
retry manager-a[o

**Comment by saivirtue** [comment]
retry manager-api

**Comment by saivirtue** [comment]
retry sshConnector

**Pull request merged**